libs/zzz/assets-data/src/executors/gen-assets-data/schema.d.ts

```
@@ -0,0 +1 @@
+export interface GenAssetsDataExecutorSchema {} // eslint-disable-line
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)